broker/server.js

```diff
@@ -11,6 +11,11 @@ const headers = {
   "content-type": "application/fhir+json",
 };
 
+//Health check endpoint
+app.get("/test", (req, res) => {
```

Comment:
I'm thinking the route should reflect the health check rather than "test" which is ambiguous to me. Maybe `/healthcheck` or `/healthstatus` or `brokerhealthstatus` to keep in line with this being a health check up?